\chapter{Introduction}
\label{ch:Introduction'}

The timely and accurate replication of DNA is a critical process in maintaining genetic integrity in cellular life.
In simple cells (called prokaryotes), the process used to replicate the genome (called the ``replication program'') is well understood.
Starting at a sequence-defined location (the ``origin''), the double-stranded DNA (dsDNA) of the prokaryote genome separates into two single-stranded DNA (ssDNA) segments.
Complex biological machinery travel bidirectionally from the origin, separating the dsDNA into growing ssDNA segments which are used as templates for the creation of two copies of the original genome.
The machinery between replicated and non-replicated DNA, called ``forks,'' continue to propagate through the genome until it has been entirely separated and replicated.
Prokaryotic organisms have such small genomes that replication can be completed using a single origin~\cite{MolecularCellBiology}.
With this replication program, an \emph{E.~coli} bacterium can replicate its entire genome in about 40 minutes\footnote{Appendix~\ref{ap:EColi}}.

More complex organisms (called eukaryotes), have genomes that are approximately 1000 times longer and have forks that propagate about 10 times slower than prokaryotes.
The replication forks from a single origin would take about $10^4$ time as long to fully separate the dsDNA for replication.
Many human cells replicate in about 24 hours, during 8 of those hours, the DNA is replicated~\cite{CellMolApproach}.
This is much less than $10^4$ times as long as the 40 minute replication time of \emph{E.~coli}.
Therefore, a single origin cannot be solely responsible for the replication of eukaryotic DNA.
Eukaryotic DNA is replicated using a parallel process, with many origins along the genome that fire stochastically~\cite{eukaryotereview}.

Using many origins in the replication program creates several non-obvious issues that must be addressed for the program to work effectively.
One such issue is the existence of separate replicated regions during the replication process.
When the forks of two neighbouring replicated regions meet, the two coalesce into a single, larger replicated region.
Another hurdle the program must overcome is the need to coordinate multiple origin activation events that are driven by stochastic processes.
In other words, the program must be able to control the replication program such that the stochastic events that drive replication do not cause errors that can potentially harm the organism.~\cite{eukaryotereview}


	\section{The Cell Cycle}
	\label{sec:CellCycle}
	
	The cell cycle defines the steps taken during cellular reproduction.
	The cell cycle can be divided into four phases (see Figure~\ref{fig:CellCycle})~\cite{MolecularCellBiology,CellMolApproach}:
	The first gap (G1) phase, the synthesis (S) phase, the second gap (G2) phase, and the mitosis (M) phase.
	The first two phases of the cell cycle (G1 and S) contain key processes for DNA replication.
	The third phase of the cell cycle (G2) primarily acts as a buffer to ensure complete DNA replication.
	During the fourth phase of the cell cycle (M) the cell physical divides into two daughter cells.
	
	\begin{SCfigure}[1][thb]
		\includegraphics[width=0.45\textwidth]{images/CellCycle.png}
		\caption[Cell Cycle]{\label{fig:CellCycle} The complete cell cycle is made up of four phases: 
			The Mitosis (M) phase, when a mother cell separates into two daughter cells. 
			The first Gap (G1) phase, when the daughter cell undergoes growth and chemical preparation for DNA replication.
			The Synthesis (S) phase, when the DNA is replicated.
			And the second Gap (G2) phase which acts as a buffer to ensure complete replication before the M phase.}
	\end{SCfigure}
	
	
		\subsection{The G1 Phase}
		\label{subsec:G1Phase}
		
		As with any cycle, the cell cycle begins at the end of the previous cell cycle.
		The G1 phase begins early in the life of a daughter cell, after the mother cell has divided in the preceding M phase.
		During this time, the cell grows and, more importantly, an important chemical process called licensing is carried out to prepare for replication during S phase.
		
		Licensing occurs at the origin recognition complex (ORC), as seen in Figure~\ref{fig:EarlyReplicationConcept}:A.
		The ORC is made up of a single group of six proteins that bind to the DNA at an origin~\cite{ORC}.
		Two additional proteins (Cdc6 and Cdt1; left out of Figure~\ref{fig:EarlyReplicationConcept}:A) assist the ORC in recruiting minichromosome maintenance (MCM) 2-7 hexamer rings onto the DNA~\cite{DNARepInitiation}.
		Loaded hexamers form pairs oriented away from each other~\cite{MCMPairs}, each such pair will later be referred to as potential initiators, or just initiators.
		After licensing, the resulting set of proteins associated with the origin is call the pre-replication complex (pre-RC).
		Licensing is suppressed during the S and G2 phases by cyclin-dependent kinases, this effectively limits the cell cycle to a single replication event.~\cite{MolecularCellBiology}
		
		
		\subsection{The S Phase}
		\label{subsec:SPhase}
		
		The second phase in the cell cycle is the S phase.
		After licensing is completed in the G1 phase, the division of the genome into two identical copies occurs during the S phase.
		There are three main processes that happen during the S phase: initiation, elongation, and coalescence, shown in Figure~\ref{fig:EarlyReplicationConcept}:B.
		
		An origin initiates (or activates, or fires) during S phase when five other proteins bind to each in a pair of MCM2-7 rings:
		Cdc45 and the tetrameric GINS complex (only Cdc45 is shown in Figure~\ref{fig:EarlyReplicationConcept}:A).
		The total system of proteins is called the CMG complex (Cdc45, MCM2-7, GINS complex) and composes a helicase which traverses the genome during the S phase.
		As an origin fires, the pre-RC disassembles and the activated pair of helicases unwind and separate the double helix of the double-stranded DNA (dsDNA) into two complementary single-stranded DNA (ssDNA) chains.~\cite{GINSComplex}
		
		After an origin has been initiated, there is a small region of ssDNA bounded on either side by the CMG complex helicases.
		The points where the dsDNA is separated into two ssDNA chains are called replication forks (or just ``forks'').
		Elongation is the process by which the replication forks, with the help of the biological machinery stored in the CMG complex~\cite{PurifiedProteins}, propagate bidirectionally from the origin, separating the dsDNA.
		As the forks propagate, DNA polymerases bind with the ssDNA between them.
		DNA polymerases use the ssDNA as a template and backbone for synthesizing dsDNA; essentially it adds the missing half back onto the separated strand.
		DNA polymerase can only propagate in the $3'$ direction.
		This poses a problem, because the two strands in dsDNA are oriented in opposing directions, so the polymerase can only smoothly traverse one of the ssDNA chains (the ``leading strand'') at each fork.
		On the other strand (the ``lagging strand''), the polymerase ``stutters:''
		It replicates a small region in the direction opposite to that of fork propagation until it hits a replicated region, leapfrogs over and past that region in the direction of fork propagation, and repeats.
		These small fragments are called Okizaki fragments.
		On the lagging strand, the Okizaki fragments are connected by DNA ligases.~\cite{MolecularCellBiology, CellMolApproach, OriginsReview}
		
		Finally, when two forks meet, coalescence occurs:
		The helicases disassemble, and the two regions of dsDNA are connected by DNA ligase.~\cite{MolecularCellBiology}
	
	\begin{figure}[tbh]
		\begin{center}
			\includegraphics[width=\textwidth]{images/EarlyReplicationConcept.png}
		\end{center}
			\caption[Events During Replication]{\label{fig:EarlyReplicationConcept} During replication, many processes encompassing many proteins and protein structures are carried out.
				This graphic illustrates a version of the G1 and S phases of the cell cycle containing only those parts that are necessary to understand the model present.
				\textbf{A} In the G1 phase, origins are located and licensed when the ORC recruits pairs of MCM2-7 hexamers onto the dsDNA.
					During S phase, pairs of Mcm2-7 hexamers are activated by the addition of the Cdc45 protein.
					After activation, the resulting structures become replicative forks that traverse the DNA unwinding the dsDNA allowing the DNA to be replicated.
				\textbf{B} Because of multiple origins in eukaryotes, there are several other events that happen during the S phase.
					At the start of S phase, several origins are licensed along the genome (top of image).
					As time progresses (down), origins fire independently (initiation) and replicative forks propagate along the genome (elongation).
					It is common for some origins to be passively replicated; that is, they can be replicated by the replicative fork from a neighbouring origin before firing themselves.
					At the end of the S phase, two identical and complete sets of dsDNA will be present (bottom of image).}
	\end{figure}
	
	
	\section{Origins of Replication}
	\label{sec:Origins}
	
	When prokaryotic DNA is replicated, a single origin is sufficient for a competent (i.e. timely and accurate) replicative program.
	In this case, the origin is located at a sequence specific site, and there is no need to worry about the firing-time except to ensure it is early enough for complete replication.
	However, eukaryotic DNA requires many origins of replication for a competent replicative program.
	The number of origins in the genome varies by species, from fewer than 800 in budding yeast~\cite{OriDB}, to about 100 000 in humans~\cite{OriginsReview}.
	When multiple origins exist on the genome, there are more questions one can ask about their behaviour:
	\begin{itemize}
		\item \emph{What determines the locations of the origins?}
		\item \emph{What controls the timing of origin firing?}
	\end{itemize}
	
	
		\subsection{Origin Locations}
		\label{subsec:OriginLocations}
		
		The question ``\emph{What determines the locations of the origins},'' can be difficult to answer.
		Depending on the organism, the answer can change significantly.
		In \emph{Saccharomyces cerevisiae} (budding yeast) origins are tightly bound in sequences between 11 and 17 base pairs (bp) in length and are effectively localized~\cite{ScottsPaper}.
		In \emph{Schizosaccharomyces pombe} (fission yeast) the origins are loosely bounded in 100 to 200 bp sequences~\cite{OriginsReview}.
		The region of potential licensing grows to about 200 kilo base pairs (kb) in the human genome~\cite{HumanGenome}.
		In \emph{Xenopus laevis} (African clawed frog) embryos, the origins are placed completely stochastically, with no sequence affinity at all~\cite{FrogEmbryo}.
		
		
		\subsection{Origin Firing-Times}
		\label{subsec:OriginTimes}
		
		Origin initiation is a chemical process, and therefore is governed in part by the diffusive flow of chemicals within the cell.
		In prokaryotes, with only a single origin, firing-time isn't very interesting:
		As long as the origin fires early enough, the DNA will be reproduced successfully.
		In eukaryotes, however, the cooperation of multiple origins can lead to interesting timing dynamics.
		
		
		
		
		\subsection{Budding Yeast}
		\label{subsec:BuddingYeast}
		
		For the research presented, \emph{S. cerevisiae} (budding yeast) was considered.
		Budding yeast is an ideal model species because, unlike the other examples provided, the origins of \emph{S. cerevisiae} are localized.
		In each cell cycle, origins of budding yeast may be licensed only in very narrow windows.
		These windows are defined by specific sequences in the genome, called autonomously replicating sequences (ARS) elements, have been identified and cataloged\footnote{an online database can be found at http://cerevisiae.oridb.org/} \cite{OriDB}.
		Thus, by choosing budding yeast as the model species, the potential stochasticity in origin locations has conveniently been removed from consideration.
		
		Previous studies of the firing-times of individual origins in budding yeast have found evidence that the process has some driving force~\cite{ScottsPaper,StochasticTermination}.
		Both studies measured the average firing-time and the spread in firing-times for each origin in budding yeast and discovered a correlation between them.
		Essentially, origins that tend to fire early have narrowly defined fire-times, while those that tend to fire late have loosely defined firing times.
		This trend implies the existence of a mechanism that strongly controls the fire-time of origins at the start of S phase, but loses its potency as S phase progresses.
		
		A theory for what physical components of the replicative machinery create this timing mechanism will be discussed in detail later [\textbf{Section number or name?}].
		
		
	\section{Modelling Replication}
	\label{sec:Modelling}
	
	To recap, DNA replication begins at origins which, in budding yeast, are localized spatially but whose firing-times exhibit stochasticity.
	Once an origin has fired, replication forks traverse the DNA bidirectionally, enclosing a growing region of replicated DNA between them.
	When two regions of replicated DNA meet, they coalesce into one larger region.
	
	This process can very easily be mapped to a crystallization process in one-dimension (Fig.~\ref{fig:CrystalVsReplication}):
	Crystallization starts when the crystal nucleates at nucleation sites, which map to origins of replication.
	From nucleation sites, the crystal grows bidirectionally, and the crystal domain is surrounded by boundaries that can be mapped to the replicative forks.
	Finally, when two crystal regions meet, they coalesce into a larger region, which matches the coalescence of neighbouring regions of replicated DNA.
	This mapping of DNA replication to crystal growth means that one can use mathematics already developed, instead of reinventing the wheel.
	
	\begin{figure}[tbh]
		\begin{center}
			\includegraphics[width=\textwidth]{Images/CrystalVsReplication.png}
		\end{center}
			\caption[Comparing Crystallization with Replication]{\label{fig:CrystalVsReplication} Comparison between the one-dimensional KJMA crystallization model and the described replication model.
				At left is the one-dimensional crystallization process, with circles representing nucleation sites (no arrow), or the propagating crystal boundaries (arrows in direction of propagation).
				Right is the replication process as described, see Fig.~\ref{fig:EarlyReplicationConcept} for a full description.}
	\end{figure}
	
	Early in the 20th century the KJMA model was developed as a stochastic model that describes crystallization growth in three dimensions\cite{Kolmogorov, JohnsonAndMehl, AvramiI, AvramiII, AvramiIII}.
	Since its inception, the KJMA model has been used for many studies that range from phase transition kinetics~\cite{AlloyPhaseTransitions} to R{\'e}nyi's car-parking problem in one dimension~\cite{CarParking}.
	With the wealth of work done in the KJMA, it shouldn't be surprising that in 2002 J.~Herrick \emph{et al.} successfully synthesized a KJMA-like model of DNA replication when analyzing experiments done on \emph{X.~laevis}~\cite{KJMA2002}.
	In 2005, the KJMA-like model was expanded, and a formalism was developed which can be used to infer the replication profile of a genome given a set of parameters that describe the speed of replication forks and the origins' locations in time and space~\cite{KJMA1, KJMA2}.
	
	One of the quantities that can be inferred using the KJMA formalism is the replicated fraction, $f(x,t)$.
	The replicated fraction can be interpreted as the probability that the genome at position $x$ has been replicated by the time $t$ after the start of S phase.
	The replicated fraction is an important quantity that will be discussed more later.
	
	
	\section{Experiments Measuring Replication}
	\label{sec:Experiments}
	
	So far, the topics covered have been conceptual and theoretical.
	However, the theory describing DNA replication has been driven by the results of experiment, so there is a question that should be addressed:
	\emph{How can DNA replication be observed experimentally?}
	As it turns out, there are several techniques available to scientists that answer this question, including flow cytometry~\cite{DeepSeq}, DNA combing~\cite{DNACombing}, microarray~\cite{MicroarrayReview, McCuneMicroArray}, and sequencing experiments~\cite{DeepSeq}.
	These experiments can be separated into categories based on probing replication primarily spatially or temporally.
	
	There are two approaches that experiments use to look at time.
	The first is to synchronize the cell cycle.
	A common way to do this is by \emph{arresting} the cell cycle, though there are other techniques~\cite{CellCycleSynch}.
	This usually entails using a chemical bath to stop the cells from progressing from one phase to another (for experiments related to DNA replication, this is generally just prior to entering the S phase).
	The main drawbacks to this approach are that it is susceptible to systematic error and that it is difficult to arrest many species of eukaryotic organisms.
	After arresting the cell cycle, another chemical bath can be used to force the entire population of cells to enter the S phase synchronously.
	The second approach is to forego arresting the cell cycle and pull samples from an asynchronous population.
	The main drawback to this method is it either creates time-average data or it relies on multiple techniques to infer the timing information.
	For this research, the first was investigated.
	
	Spatially, approaches range between two extremes: Perfect base pair resolution and no spacial information.
	At one extreme, some experiments calculate one value over the entire genome. 
	For example a common flow cytometry technique called fluorescence activated cell sorting (FACS)~\cite{SequencingReview, DeepSeq}, measures only the amount of DNA in the cell.
	Techniques such as FACS provide only limited information but are simple and fast.
	At the other extreme, other techniques such as DNA sequencing and DNA microarray are able to spatially resolve to small windows (as small as 1 kb~\cite{DeepSeq}).
	Experiments with this level of resolution are quite complex but provide tremendous insight toward understanding DNA replication.
	For this research, the second scope was investigated.
	
	Due to their impact on the research presented in this thesis, microarray experiments and sequencing experiments will be discussed in more detail.
	Both experiments were designed to maximize spacial resolution, and both can use either a synchronous or an asynchronous population.
	
	
		\subsection{Microarray Experiments}
		\label{subsec:Microarray}
		
		Microarray experiments are high-throughput experiments that count the genome of entire populations of cells simultaneously~\cite{MicroarrayReview}.
		They start with a microarray chip\footnote{actually, many chips} and a population of cells.
		The DNA of the population is extracted and hybridized with the chip, which provides a measure of the replicated fraction, $f$.
		Depending on the temporal scope of the experiment, the measured replicated fraction can be time-averaged, $f(x)$, or in the case of an arrested population, it can be the replicated fraction for a specific time, $t_i$ after the start of S phase, $f(x,t=t_i)$.
		
		Because this technique measures entire populations, microarray experiments do not provide information about cell-to-cell variability.
		More importantly, microarrays suffer from artifacts that can be challenging to overcome.
		For example, in 2008, McCune \emph{et al.} measured replication fractions that spanned only 80\% of the possible values~\cite{McCuneMicroArray}.
		
		
		\subsection{Sequencing Experiments}
		\label{subsec:Sequencing}
		
		A sequencing experiment determines the precise sequence of base pairs contained in the input segment of DNA~\cite{SequencingReview}.
		To measure the replicated fraction using sequencing, one must start with with the fully mapped genome of the organism in question.
		The DNA from a population of cells to be measured is harvested and broken into segments about 50 bp long~\cite{StochasticTermination}.
		Each segment can be sequenced and matched to the previously mapped genome.
		The normalized histogram of reads over the genome then provides the replicated fraction:
		Regions that have not replicated are measured once and regions that have replicated are measured twice.
		
		Except for the actual process of measuring how many of each segment of DNA is present in the sample, sequencing experiments and microarray experiments are very similar.
		The process of arresting cells, or not, is the same for both, as is the broad strokes of analyzing the output.
		However, sequencing experiments do not required any clever data-processing to remove artifacts like those discussed already~\cite{EndOfMicroarray}.
		Recent advances lowering the cost of sequencing have seen a transition from microarray experiments to sequencing experiments for measuring DNA replication.
		
		
	\section{Thesis map \textbf{UPDATE}}
	\label{sec:Map}

	Here I will give a brief outline of the thesis and how to traverse it. For now this will serve as a plan for the outline, but after writing this will be more of a road-map for readers than for me.
	
	\textbf{Chapter 2 - Motivation}
	should introduce the mathematics of the model more clearly.
	A discussion of the replicated fraction and how to calculate it.
	It should outline Scott's Sigmoidal Model and the observed correlation between $t_{1/2}$ and $t_{width}$.
	It will then introduce the Multiple Initiator Model, its assumptions, and its results.
	Finally, it will discuss Nick's 2014 experiment measuring loaded MCM and the potential problem it points to.
	
	\textbf{Chapter 3 - Method}
	should talk about the different tools I used to try to understand the problem and its solution.
	Phantom Nuclei Simulation.
	Monte Carlo (compare to analytic, pros and cons).
	Igor global fitting.
	
	\textbf{Chapter 4 - Results}
	talk about results (GET THESE!)
	
	\textbf{Chapter 5 - Conclusions}
	discuss the implications of my results.
	where should research go from here?
		
		
		
		
		
		
		
		
		
		