\chapter{Motivation}
\label{ch:Motivation}

Previous research into DNA replication has investigated the timing of origin initiation~\cite{ScottsPaper,Bechhoefer2012374,deMouraModel1,deMouraModel2,StochasticTermination}.
In 2010, Yang~\emph{et~al.} fit the replication fraction of an origin, $f(x=x_o,t)$, to a Sigmoidal Model to better understand origin firing time~\cite{ScottsPaper} (Sec.~\ref{sec:SigmoidalModel}).
The results (discussed in detail below) were confirmed using a different model by Hawkins~\emph{et~al.} in 2013~\cite{StochasticTermination}.

The work of Yang~\emph{et~al.} lead to the development of a second analytical model called the Multiple Initiator Model (MIM)~\cite{ScottsPaper} (Sec.~\ref{sec:MIM}).
The MIM made some reasonable assumptions to simplify the math involved.
With this mode experimental data were analyzed and conclusions about the physical mechanisms controlling DNA replication.

However, recent work performed in N. Rhind's lab [\textbf{Source this, somehow}] have shown that one of the assumptions used in the MIM may not be physically realistic (Sec.~\ref{sec:ExperimentsMIM}).
The purpose of this research is to discover the impact of this new information on the MIM.

This chapter will expand on the above story, filling in the details of the math behind the models, and explicitly stating the assumptions and measurements made.


	\section{Replicated Fraction}
	\label{sec:ReplicatedFraction}
	
	It was mentioned briefly in Sec.~\ref{sec:Modelling} and Sec.~\ref{sec:ExperimentsBasics} that the replicated fraction, $f$, can be calculated from both theoretical models and experiments.
	Thus, the replicated fraction is a quantity that can be used to analyze experimental data and to test the validity of models.
	For this reason we will now provide details about what the replicated fraction is, how it is calculated, and how we used it in our research.
	
	There are two ways to describe the replicated fraction as a function of time and space, $f(x,t)$:
	as a quantity of either a single cell, or a population of cells.
	In the single cell case, $f(x,t)$ is the probability that the sequence at position, $x$, in the genome has replicated a time, $t$, after the start of S phase.
	From a population of cells, $f(x,t)$ is the fraction of the cells in the population that have replicated at position, $x$, a time, $t$, after the start of S phase.
	It is not difficult to see the equivalency between these two descriptions, but there is a subtle difference that will become apparent soon (Sec.~\ref{sec:MIM}).
	
	
		\subsection{Qualities of the Replicated Fraction}
		\label{subsec:QualitiesReplicatedFraction}
		
		Before describing the mathematical formulae that define the replicated fraction of the KJMA-like model of DNA replication, it is valuable to build some intuition.
		In DNA sequencing experiments, the replicated fraction is measured spatially in windows about 1 kb wide, and temporally in steps of 5 minutes~\cite{StochasticTermination}.
		Budding yeast's genome is over $12\times10^3$ kb long so the there is tremendous spatial data.
		However, Budding yeast completely replicates it's DNA in less than 90 minutes~\cite{DeepSeq}, and most experiments stop performing thorough measurements after about 50 minutes~\cite{StochasticTermination,DeepSeq,McCuneMicroArray}.
		There are generally no more than 10 values for $t$ measured experimentally (indeed, the data analyzed in Sec. [\textbf{Section Reference}] has only 6).
		Fortunately, this amount of temporal data is enough to build an intuitive understanding of the replicated fraction and to analyze it mathematically.
		
		Figure~\ref{fig:ReplicatedFractionExample} shows an example set of replicated fraction data.
		The data comes from measurements done on chromosome IV of budding yeast by Hawkins~\emph{et~al.}~\cite{StochasticTermination}.
		The perceptive reader may notice a few things:
		there are gaps in the spatial data;
		the replicated fraction ranges lower than zero and higher than 1;
		and some regions of the genome replicate faster than others.
		
		\begin{figure}[tbh]
			\begin{center}
				\includegraphics[width=\textwidth]{Images/CHR4Exp.png}
			\end{center}
				\caption[Budding yeast chromosome IV replicated fraction]{\label{fig:ReplicatedFractionExample} Example graph of replicated fraction.
					Data from chromosome IV of budding yeast as measured by Hawkins~\emph{et~al.}~\cite{StochasticTermination}.	
					X-axis represents the spatial organization of the genome as if it had been stretched out straight.
					Y-axis is the replicated fraction.
					6 data sets present as measurements made at various times throughout S phase.
					[\textbf{Do I need permission for this?}]
				}
		\end{figure}
		
		The gaps in Fig.~\ref{fig:ReplicatedFractionExample} exist because of a limitation of the sequencing experiment used to gather this data.
		Sequencing experiments match small chunks of DNA to the fully mapped genome (Sec.~\ref{subsec:Sequencing}).
		The budding yeast genome contains repeated patterns: sequences longer than 50 bp that appear more than once~[\textbf{FIND SOURCE}].
		When a chunk of DNA that is enclosed in one of these patterns is measured, it is not counted because it cannot be uniquely located.
		
		The replicated fraction in Fig.~\ref{fig:ReplicatedFractionExample} has a larger range than is theoretically possible.
		This is not because some regions are absent or replicated doubly, but because of inaccuracies in the experimental procedure.
		In the presented data, this comes from two assumptions made: first that the measured sequences were evenly distributed spatially, and second that all cells have the same average replicated fraction at the time measured, $f(t=t_i)$~\cite{StochasticTermination}.
		In the experiment by Hawkins~\emph{et~al.}, they acquired 10-25 million 50 bp reads.
		If all of those reads were spatially uniform, that's a read of 50-100 full genomes.
		However, the only factor that pushes the reads toward spatial uniformity is probability, so there will be some regions that are read more and some that are read less.
		Additionally, Hawkins~\emph{et~al.} normalized the measured replicated fraction by ensuring the average replicated fraction, $f(t=t_i)$, was equal to the replicated fraction measured using FACS on the bulk sample.
		This normalization assumes (reasonably) that the ``100 full cells'' measured have the same average replicated fraction as the population measured with FACS.
		These two processes combine to increase the measured range for $f$ outside of numbers that are theoretically possible.
		
		The most important observation is the fact that some regions of the genome start replicating much earlier than others.
		This can be seen in the peaks in Figure~\ref{fig:ReplicatedFractionExample}, for example at $x \approx 910$.
		Because replication starts at an origin and propagates outward, peaks in the replicated fraction should be centered on origins.
		Additionally, early origins should create peaks earlier in the program, and late origins should create peaks later in the program.
		
		
		\subsection{Calculating Replicated Fraction from the KJMA Formalism}
		\label{subsec:KJMA}
		
		The replication program is defined by the origins.
		Specifically, the spatial and temporal organization of the origins has a large impact on replication.
		The speed at which the replicative forks propagate also play a roll in determining the replicative program.
		Based on the work of Jun~\emph{et~al.}~\cite{KJMA1}, here we outline the mathematical process of calculating the replicated fraction from a set of parameters describing the origins of replication and the replicative forks.
		
		We define the rate of initiation, $I(x,t)$, at an unreplicated position, $x$, and time, $t$, after the start of S phase.
		Of course, initiation can only happen at origins of replication, and in budding yeast origins are localized at known locations~\cite{OriDB}, labeled $x_i$.
		Therefore we define the rate of initiation at origin $i$ as $I_i(x,t)=\delta(x-x_i)I_o(x,t)$, where $\delta(x)$ is the Dirac $\delta$ function.
		Finally, we define the rate of initiation as $I(x,t) = \sum\limits_i I_i(x,t)$.
		
		Given the function $I(x,t)$, we can infer the replicated fraction, $f(x,t)$, at a position $x$ a time, $t$, after the start of S phase:
		\begin{equation} \label{eq:RepFromI}
			f\left( x,t\right) = 1 - \prod_\Delta\left[1-I\left( x^\prime,t^\prime\right)\Delta x^\prime\Delta t^\prime\right] \text{ ,}
		\end{equation}
		where the product is over the past triangle shown in Fig.~[\textbf{Make/TakeFigure}].
		In words, Eq.~\ref{eq:RepFromI} says that the probability that the genome at position $x$ has been replicated is one minus the probability that no origin has fired long enough in the past to have a replicative fork pass over position $x$.
		In the limit $\Delta x\rightarrow0$ and $\Delta t\rightarrow0$, Eq.~\ref{eq:RepFromI} becomes
		\begin{equation} \label{eq:RepFromIexp}
			f\left( x,t\right) = 1 - \exp\left[-\iint\limits_\Delta dx^\prime dt^\prime I\left( x^\prime,t^\prime\right)\right] \text{ .}
		\end{equation}
		
		Now, it is possible to define a new quantity, $g(\Delta x_p,t)$, that is a local measure of origin firing:
		\begin{equation} \label{eq:LocalOriginFiring}
			g\left(\Delta x_p,t\right) = \int\limits_{x_p}^{x_{p+1}}\delta\left( x-x_i\right) dx \int\limits_o^t I_0\left( x,t^\prime\right) dt^\prime
		\end{equation}
		over the region $[x_p, x_{p+1})$ of a genome of length, $L$, discretized into $M$ segments;
		\begin{align}
			\Delta x = \frac{L}{M} \qquad\qquad x_p = p\left(\Delta x\right) \qquad\qquad p = 0, 1, 2, \ldots , M-1 \text{ .}
		\end{align}
		$g(\Delta x_p,t)=0$ if there are no origins enclosed in $\Delta x_p$ because initiation will only occur at an origin.
		Thus, we can replace the double integral in Eq.~\ref{eq:RepFromIexp} with the function $g(\Delta x_p,t)$ and arrive at
		\begin{equation} \label{eq:RepFromG}
			f\left( x,t\right) = 1 - \exp\left[ - \sum\limits_{p=0}^{M-1}g\left(\Delta x_p,t-\frac{\left| x-x_p \right|}{v}\right)\right] \text{ ,}
		\end{equation}
		where $v$ is the speed of replicative forks, $\Delta x_p$ is the $p^\text{th}$ interval, $x_p$ is the $p^\text{th}$ position, and $\left| x-x_p \right|/v$ is the time at the edge of the triangle in Fig.~[\textbf{Make/Take}].
		
		Recognizing that $g(\Delta x_p,t)$ represents the initiation rate of budding yeast, we can constrain it to better represent the biological system:
		First, we constrain $g$ such that replication can not happen before the start of S phase; $g(\Delta x_p,t<0)=0$.
		Second, we constrain the initiation rate such that it cannot be negative. Because of the definition in Eq.~\ref{eq:LocalOriginFiring}, this constrains $g$ as well: $\frac{d}{dt}g(\Delta x_p,t)\geq 0$.
		Third, as a consequence of the first two constrains, $g(\Delta x_p,t)\geq 0$.
		
		Finally, we derive the cumulative initiation probability, $\Phi(x_p,t)$, from $g(\Delta x_p,t)$ using a Poisson-process calculation~[\textbf{Cite Spikes}]:
		\begin{equation} \label{eq:PhiFromG}
			\Phi\left( x_p,t\right) = 1 - e^{g\left(\Delta x_p,t\right)} \text{ .}
		\end{equation}
		The cumulative initiation distribution is an important quantity which will be revisited below {Sec.~\ref{sec:MIM}).
		Note that $\Phi(x_p,t)$ is a general function that can be defined throughout the genome, but in the case of budding yeast is nonzero only for values of $x_p$ that coincide with origins.


	\section{The Sigmoidal Model}
	\label{sec:SigmoidalModel}
	
	The sigmoidal model is an approach to analyzing DNA replication that investigates the replicated fraction at each origin, $f(x=x_i,t)$.
	Developed by S. Yang as part of his PhD thesis, this model assumes that the functional form of $f(x=x_i,t)$ is a sigmoidal function that has a range from zero to one and that is defined by three parameters at each origin, $i$, on the genome~\cite{ScottsPaper,ScottsThesis}.
	
	Figure~\ref{fig:SigmoidalModel} illustrates the approach of the sigmoidal model and Yang's results.
	First, extract the replicated fraction at an origin, $f(x=x_i,t)$, from experimental data of the entire genome, $f(x,t)$ (Fig.~\ref{fig:SigmoidalModel}A).
	For his thesis, Yang used microarray data~\cite{McCuneMicroArray}.
	Second, fit a sigmoidal curve to $f(x=x_i,t)$ (Fig.~\ref{fig:SigmoidalModel}B).
	This sigmoidal curve is parameterized to define the median replication time, $t_{\text{rep}}$, and the spread of replication times, $t_{\text{width}}$:
	\begin{equation} \label{eq:SigmoidalModel}
		f(t) = {\frac{1}{1+\left({\frac{t_{\text{rep}}}{t}}\right)^r}}\text{ ,}
	\end{equation}
	where $t_{\text{width}}$ is defined by
	\begin{equation}
		t_{\text{width}} = \left(3^{1/r}-3^{-1/r}\right)t_{\text{rep}}\text{ .}
	\end{equation}
	
	When $t_{\text{rep}}$ vs $t_{\text{width}}$ is graphed an interesting trend appears (Fig.~\ref{fig:SigmoidalModel}C).
	A strong correlation exists between $t_{\text{rep}}$ and $t_{\text{width}}$.
	The correlation means that origins that tend to fire early have very narrowly defined firing times, while origins that tend to fire late have more loosely defined firing times.
	The conclusion one can draw from this insight is that there is a mechanism that has control over origin firing time that is strong at the start of S phase but weakens as S phase progresses~\cite{ScottsThesis}.
	From this theory, a new model was developed called the Multiple Initiator Model (MIM).
	
	\begin{figure}[tbh]
		\begin{center}
			\includegraphics[width=.82\textwidth]{Images/ScottFig3-1.pdf}
		\end{center}
			\caption[Sigmoidal model]{\label{fig:SigmoidalModel} Schematic of the Sigmoidal Model process and results from analysis of budding yeast data.
				\textbf{A} Sample replicated fraction: smoothed data from microarray measurements of Chromosome I (solid lines)~\cite{McCuneMicroArray}.
				The black triangles indicate the locations of previously identified origins~\cite{OriginLocations}.
				To fit the sigmoidal model, take only the replicated fraction at an origin (grey region).
				\textbf{B} Fit Eq.~\ref{eq:SigmoidalModel} to the extracted replicated fraction at an origin.
				The fitted function has parameters, $t_{\text{rep}}$ and $t_{\text{width}}$, which are shown.
				\textbf{C} After performing this process on every identified origin, a correlation is identified between these two parameters.
				Figure reproduced with permission from S.~Yang [PhD Thesis]~\cite{ScottsThesis} (Copyright~\textcopyright~2012)}
	\end{figure}
	
	
	\section{The Multiple Initiator Model}
	\label{sec:MIM}
	
	The MIM, in its simplest terms, says that each origin has some number of unique chances to be initiated at any given time~\cite{ScottsThesis}.
	If each of these chances are equally weighted, then origins with large numbers of chances should tend to fire earlier than origins with few chances.
	One hypothesis for the physical mechanism that creates these unique chanced to initiate is the MCM2-7 (initiators) pairs loaded at each origin.
	Effectively, origins with more initiators loaded will tend to fire earlier in S phase than origins with fewer pairs.
	However, it is important to note that factors besides the number of loaded initiators pairs can impact the relative firing times of origins.
	For example, the chromatin structure (the three dimensional organization of the genome) is believed to play a roll in controlling origin firing times~\cite{Chromatin}.
	
		\subsection{MIM Basics}
		\label{subsec:MIMBasics}
		During licensing, the ORC can load MCM2-7 hexamers in excess~\cite{MultiMCM}.
		We believe this the loading of initiators is a Poisson-process:
		MCM2-7 hexamers are loaded at an origin individually with some probability determined by the affinity of that origin.
		We assume that the average number of initiators loaded at the $i^{\text{th}}$ origin, $n_i$, is a fixed quantity over many cell cycles.
		However, the actual number of initiators, $N_i$, can change between cell cycles and is believed to be exponentially distributed as a result of the Poisson-process~[\textbf{FindSource}].
		
		During S phase, the initiators are activated by the addition of Cdc45 and the GINS complex.
		The MIM assumes that each initiator has the same cumulative probability of firing as time progresses through S phase, given by
		\begin{equation}\label{eq:CPDInitiator}
			\Phi_0(t) = \frac{1}{1+\left(\frac{t^*_{1/2}}{t}\right)^{r^*}}\text{ .}
		\end{equation}
		Where $t^*_{1/2}$ is the median firing-time for a single initiator and $r^*$ is the rate of increase.
		These variables are global, defining the behaviour of every initiator on the genome.
		Using this assumption, then the cumulative probability that an origin with $N$ loaded initiators is given by
		\begin{equation} \label{eq:CPDEffectiveN}
			\Phi_{\text{eff}}(t,N) = 1 - \left[1 - \Phi_0(t)\right]^N\text{ .}
		\end{equation}
		
		Using Eq.~\ref{eq:CPDEffectiveN}, the replicated fraction can be inferred from a set of global parameters (fork velocity, time, $t^*_{1/2}$, $r^*$, and two parameters defining the noise in the experimental data) and two parameters per origin (its position on the genome, and the number of initiators it loads).
		We start by calculating the effective cumulative firing time distribution, $\Phi_{i\text{eff}}(x,t,N_i)$ for each origin, $i$.
		Next we invert Eq.~\ref{eq:PhiFromG},
		\begin{equation}
			\ln \left( 1- \Phi_{i\text{eff}}(x,t,N_i)\right) = - g(\Delta x_p, t) \text{ ,}
		\end{equation}
		to get the measure of initiation of origin $i$.
		Summing the results over every origin,
		\begin{equation}
			\sum\limits_{\text{all origins }i}\Phi_{i\text{eff}}(x,t,N_i) = - \sum\limits_{p=0}^{M-1} g(\Delta x_p,t) \text{ ,}
		\end{equation}
		gives us the initiation rate for the entire genome.
		Finally, if we've been clever with how we track this data spatially, we can use this as the exponent in Eq.~\ref{eq:RepFromG}.
		Using this process, Yang fit the parameters listed above to experimental data in his PhD work~\cite{ScottsThesis}.
	
		Equation~\ref{eq:CPDEffectiveN} calculates the effective cumulative probability distribution of an origin with $N$ loaded initiators.
		This is a property of a single-cell, with a single value for $N$.
		If the number of loaded initiators at an origin does not change between cell cycles (i.e. $N=n$), then Eq.~\ref{eq:CPDEffectiveN} applies to large cell populations as effectively as a single cell.
		However, it does not take into account any variability in $N$ between cell cycles.
		In other words, $\Phi_{\text{eff}}$ cannot be calculated from population data the same way:
		\begin{equation} \label{eq:CPDEffectivenTemp}
			\Phi_{\text{eff}}(t,n) \neq 1 - \left[1 - \Phi_0(t)\right]^n\text{ .}
		\end{equation}
		
		
		\subsection{Accounting for variability in $N$}
		\label{subsec:VariableN}
		
		Given the many factors that effect the ORC's ability to load MCM2-7 initiators onto DNA~\cite{MultiMCM}, it is reasonable to assume that the number of initiators will not be the same between cell cycles.
		Thus, the fact that Eq.~\ref{eq:CPDEffectivenTemp} will only be an equality when $N=n$ must be addressed.
		
		The assumption made by S.~Yang in his thesis is that initiators are loaded as a Poisson-process.
		This means in a large population of cells, if a particular origin has a mean number of initiators, $n$, the standard deviation of $N$ will be $\sqrt{n}$~[\textbf{does this need a citation?}].
		Therefore, as $n$ grows, the relative differences in $N$ within the population shrinks.
		Thus, for large enough $n$, $N\approx n$, so Eq.~\ref{eq:CPDEffectivenTemp} becomes
		\begin{equation} \label{eq:PhiEffective}
			\Phi_{\text{eff}}(t,n) = \lim_{n\to\infty} 1 - \left[1 - \Phi_0(t)\right]^n\text{ .}
		\end{equation}
		
		A new problem arises from Eq.~\ref{eq:PhiEffective}:
		It is not the case that an infinite number of initiators are loaded at an origin \textsuperscript{[citation needed]}, so it is necessary to determine at what value $n$ is ``large enough.''
		When the MIM was first developed, this threshold was set at $n=$ \textbf{FIND THIS}.
		Consequently, assuming that for many origins $n>$ \textbf{FIND THIS}, the parameters of the MIM were fit to microarray data.
		The results showed that for the majority of origins, the assumption was true~\cite{ScottsThesis}.
		Note that those origins where $n<$ \textbf{FIND THIS} are late-firing origins because of their low $n$, and they have negligible impact on the replication process.
		
	\section{Experimental Measure of the Number of Initiators}
	\label{sec:ExperimentsMIM}
	
	The MIM presents a strong theory about the physical mechanism controlling origin fire-times during S phase.
	A part of the strength of this theory is it can be checked experimentally by counting the number of initiators loaded at an origin, and in 2013 S.~P.~Das~\emph{et~al.} did just that~[\textbf{Figure out how to cite this}].
	
	Das~\emph{et~al.} made several measurements to test the strength of the MIM.
	First, they measured the relative number of initiators loaded at every origin; according to the MIM, origins that fire earlier should have more initiators than those that fire later.
	Second, they measured the effect of reducing the number of initiators at an origin; according to the MIM, reducing the number of loaded initiators should delay the mean firing time of that origin.
	Third, they measured the average number of initiators loaded on a particular early origin; according to the MIM, an early-firing origin should have many initiators loaded on average.
	
		\subsection{Relative Number of Initiators}
		\label{subsec:RelativeNo}
		
		To measure the relative number of initiators, Das~\emph{et~al.} used ChIP-seq in a population of G1-arrested cells.
		ChIP-seq experiments measure the interaction between proteins and DNA~[\textbf{Find Source}].
		In this case, Das~\emph{et~al.} prepared the experiment so that the output provided a measure of the relative number of MCM proteins loaded throughout the genome.
		Figure~[\textbf{Das Figure 1A}] shows the measure through Chromosome X of budding yeast, and peaks align nicely with origins identified in OriDB~\cite{OriDB}.
		Figure~[\textbf{Das Figure 1B}] shows the relative number of MCM2-7 hexamers loaded at an origin vs the $n$ value predicted from the MIM in 2010~\cite{ScottsPaper}.
		After ignoring those origins that are believed to fire late due to their location in chromatin structure (Blue origins in Fig.~[\textbf{Das Figure 1B}])~\cite{Chromatin}, a correlation between the number of initiators and the theoretical parameter $n$ is observed.
		This measurement nicely confirms the first prediction made by the MIM.
		
		
		\subsection{Suppressing the Loading of Initiators}
		\label{subsec:SuppressingInitiators}
		
		To measure the effect of suppressing the loading of initiators on the replication program, Das~\emph{et~al.} measured the replication profile of ARS1 and the ARS1-$\Delta$B2 mutant.
		The ARS1 origin is known to be fairly early firing~\cite{OriDB}, and should therefore have a relatively high number of initiators, and the ARS1-$\Delta$B2 mutant reduces MCM2-7 loading~[\textbf{AddSourceToDB}]%\cite{ARS1Mutant}.
		The expectation, based on the MIM, is that the mutant ARS1-$\Delta$B2 will have a later mean firing time due to the reduced number of initiators loaded.
		By measuring the replicated fraction of cells with both wild-type and mutant ARS1 origins independently, Das~\emph{et~al.} saw a marked (13 minute) delay in the average replication timing of ARS1 caused by the $\Delta$B2 mutation (Fig.~[\textbf{Das Figure 3C}]).
		
		
		\subsection{Number of Initiators Loaded}
		\label{subsec:NoInitiatorsLoaded}
		
		Das~\emph{et~al.} engineered several special plasmids to measure the average number of initiators loaded at an early firing origin,.
		A plasmid is a small loop of dsDNA that is separate from the genome and that is able to replicate independently~[\textbf{Find Source}].
		On each plasmid was engineered one of a selection of origins (each origin was contained in its own population).
		One of the six proteins in the ORC and one of the six proteins in the MCM2-7 hexamer were tagged so their relative average numbers could be measured with western blotting.
		In order to calculate the absolute average numbers of MCM2-7 hexamers and ORCs, the measured values were normalized using a zinc-finger.
		Zif268 will bind to a specific 10 bp sequence of DNA (which was included in the plasmid) with sub-nanomolar affinity~\cite{ZincFingers}.
		From a population of G1 phase arrested cells, the engineered plasmids were extracted and the relative number of ORCs, MCM2-7s and zinc-fingers were measured and normalized such that the average number of zinc-fingers was one.
		
		Of the origins investigated by Das~\emph{et~al.}, ARS1 is illustrative, and their measurements for ARS1 are shown in Fig.~[\textbf{Das Figure 2B}].
		The figure shows the average number of loaded MCM2-7 hexamers (one initiator is a pair of these), and the average number of loaded ORCs during G1 arrest and G2 arrest.
		Notably, Das~\emph{et~al.} measured $n\approx 3$ on the ARS1 origin during G1 arrest, compared to $n\approx 24$ predicted by the MIM~\cite{ScottsPaper}.
		It is important to restate that $n$ does not necessarily equal the number of initiators, but is proportional to the number of unique chances an origin has to fire.
		So, the fact that the number measured by Das~\emph{et~al.} does not match that predicted by the MIM isn't immediately troublesome.
		However, the fact that the number of initiators is so slow, means that the number (which must be an integer value on the single-cell scale), may vary greatly between cells in a population.
		The question we set out to answer with this research is \emph{how does variability in the initiation factor affect the replication program?}










































