\chapter{Motivation}
\label{ch:Motivation}

Previous work on quantitative modelling of DNA replication has investigated the timing of origin initiation~\cite{ScottsPaper,deMouraModel1,StochasticTermination,Goldar2009,OriginTimingReview}.
In 2010, Yang~\emph{et~al.} developed the ``Sigmoidal Model,'' which uses three parameters per origin (position, median firing time, and spread in firing time) to describe the replication program of budding yeast~\cite{ScottsPaper}.
After fitting these parameters to microarray data, the authors observed a correlation between the median firing time and spread in firing time.
This result (discussed in detail in Sec.~\ref{sec:SigmoidalModel}) was confirmed by Hawkins~\emph{et~al.} when they used a similar model to analyze sequencing data in 2013~\cite{StochasticTermination}.

The work of Yang~\emph{et~al.} led to the development of a second analytical model, ``the Multiple Initiator Model'' (MIM)~\cite{ScottsPaper}, which we present in Sec.~\ref{sec:MIM}.
The MIM proposes a biological hypothesis that explains the observed correlation between median firing time and spread in firing time.
The benefit of the MIM over the Sigmoidal Model is that the MIM uses only two parameters per origin to define the replicated fraction, effectively removing one third of the parameters from the model.

However, recent work performed in N. Rhind's lab~\cite{Rhind} has shown that one part of the scenario assumed in the MIM may not be biologically realistic (Sec.~\ref{sec:ExperimentsMIM}).
The purpose of this thesis is to explore the impact of these new experimental data on the MIM and show that \textbf{conclusion}.

This chapter will expand on the above story.


	\section{Replicated Fraction}
	\label{sec:ReplicatedFraction}
	
	In Secs.~\ref{sec:Modelling} and~\ref{sec:ExperimentsBasics}, we saw that the replicated fraction, $f$, can be calculated from both theoretical models and experiments.
	The replicated fraction as a function of time and space, $f(x,t)$, can be interpreted two ways:
	as describing either a single cell, or a population of cells.
	In the single-cell case, $f(x,t)$ is interpreted as the probability that the sequence at position $x$ in the genome has replicated by a time $t$ after the start of S phase.
	For a population of cells, $f(x,t)$ represents the fraction of cells in the population that have replicated at position $x$ by a time $t$ after the start of S phase.
	Although the two interpretations of $f$ might seem equivalent, we will see in Sec.~\ref{sec:MIM} that they are subtly different.
	Both definitions lead to a function that has values ranging from zero (no replication has occurred), to one (replication has certainly occurred).
	
	
		\subsection{Qualities of the Replicated Fraction}
		\label{subsec:QualitiesReplicatedFraction}
		
		Before we describe in detail the KJMA-like model of DNA replication, we will build some valuable intuition.
		In DNA sequencing experiments, the replicated fraction is measured spatially in windows about 1 kb wide and temporally in steps of 5 minutes~\cite{StochasticTermination}.
		Since the budding yeast genome has about 800 origins of replication, and is about $12\time10^3$ kb long, origins are, on average, spaced every 15 kb~\cite{OriDB,BuddingYeastSequence}.
		Thus, a spatial resolution of 1kb is narrow enough to uniquely identify and observe many origins individually. 
		However, there are generally no more than ten time points measured experimentally~\cite{StochasticTermination,DeepSeq,McCuneMicroArray}.
		(Indeed, the data analyzed in Sec.~[\textbf{Section Reference}] has only six.)
		Fortunately, this amount of temporal data is enough to infer the important features of the replication program.
		
		Figure~\ref{fig:ReplicatedFractionExample} shows an example set of replicated fraction data.
		The data comes from measurements done on Chromosome IV of budding yeast by Hawkins~\emph{et~al.}~\cite{StochasticTermination}.
		The reader may notice a few features:
		there are gaps in the spatial data;
		the replicated fraction ranges lower than zero and higher than one;
		and some regions of the genome replicate faster than others.
		
		\begin{figure}[tbh]
			\begin{center}
				\includegraphics[width=\textwidth]{Images/CHR4Exp.png}
			\end{center}
				\caption[Budding yeast Chromosome IV replicated fraction]{\label{fig:ReplicatedFractionExample} Example graph of replicated fraction.
					Data from chromosome IV of budding yeast as measured by Hawkins~\emph{et~al.} (\cite{StochasticTermination} supplementary data).
					x-axis represents the spatial organization of the genome as if it had been stretched out straight.
					y-axis is the replicated fraction.
					Six time points.
				}
		\end{figure}
		
		The gaps in Fig.~\ref{fig:ReplicatedFractionExample} exist because of a limitation of the sequencing experiment used to gather this data.
		Sequencing experiments match short sequences of DNA to the fully mapped genome (Sec.~\ref{subsec:Sequencing}).
		The budding yeast genome contains repeated patterns: sequences longer than 50 bp that appear more than once~[\textbf{FIND SOURCE}].
		When a sequence of DNA extracted from one of these patterns is measured, it is not counted because it cannot be uniquely located.
		
		The replicated fraction in Fig.~\ref{fig:ReplicatedFractionExample} has a range that goes below zero and above one.
		This surprising feature results from two assumptions: first, that the measured sequences were evenly distributed spatially; and, second, that all cells have the same average replicated fraction at the time measured, $f(t=t_i)$.
		In~\cite{StochasticTermination}, Hawkins~\emph{et~al.} extracted and measured about $10^7$ sequences.
		However, there is no guarantee that these sequences are evenly distributed, therefore there will be regions of the genome that are sequenced more and regions that are sequenced less.
		Additionally, Hawkins~\emph{et~al.} normalized the measured replicated fraction by setting the average replicated fraction, $f(t=t_i)$, equal to the replicated fraction measured using FACS on the bulk sample.
		This normalization assumes that the measured cells have the same average replicated fraction as the population measured with FACS.
		The error in regions that have been over-extracted or under-extracted can be exaggerated by normalizing incorrectly, leading to values of $f$ greater than one or less than zero.
		
		The most important observation is that some regions of the genome start replicating much earlier than others.
		This can be seen in the peaks in Fig.~\ref{fig:ReplicatedFractionExample}, for example at $x \approx 910$.
		Because replication starts at an origin and propagates outward, peaks in the replicated fraction imply early replication and, hence, the presence of origins.
		Additionally, early origins should create stronger peaks, and late origins should create weaker peaks.
		
		
		\subsection{Calculating Replicated Fraction from the KJMA Formalism}
		\label{subsec:KJMA}
		
		The replication program is defined by the origins through their spatial and temporal organization.
		The speed at which the replicative forks propagate also plays a role in determining the replicative program.
		Based on the work of Jun~\emph{et~al.}~\cite{KJMA1}, here we outline calculation of the replicated fraction from a set of parameters describing the origins of replication and the replicative forks.
		
		We define the rate of initiation, $I(x,t)$, to be the number of origins initiated per time per genome length at an unreplicated position $x$, and time $t$ after the start of S phase.
		Of course, initiation can happen only at origins of replication.
		In budding yeast, origins are localized at known locations~\cite{OriDB}, labeled $x_i$.
		Therefore, we define the rate of initiation at origin $i$ to be $I_i(x,t)=\delta(x-x_i)I_i(t)$, where $\delta(x)$ is the Dirac $\delta$ function.
		Finally, we define the rate of initiation to be $I(x,t) = \sum\limits_i I_i(x,t)$.
		
		\begin{figure}[tbh]
			\begin{center}
				\includegraphics[width=\textwidth]{Images/KJMA.png}
			\end{center}
				\caption[Replicated fraction from KJMA]{\label{fig:KJMA} KJMA approach to calculating $f(x,t)$.
					In order for the point at $(x,t)$ to not have been replicated, there cannot be any initiation events within the shaded triangle.
				}
		\end{figure}
		
		Given the function $I(x,t)$, we can infer the replicated fraction, $f(x,t)$, at a position $x$ a time, $t$, after the start of S phase:
		\begin{equation} \label{eq:RepFromI}
			f\left( x,t\right) = 1 - \prod_\Delta\left[1-I\left( x^\prime,t^\prime\right)\Delta x^\prime\Delta t^\prime\right] \text{ ,}
		\end{equation}
		where the product is over intervals $\Delta x^\prime \Delta t^\prime$ lying within the ``past triangle'' shown in Fig.~\ref{fig:KJMA}.
		In words, Eq.~\ref{eq:RepFromI} says that the probability that the genome at position $x$ has been replicated is one minus the probability that no origin has fired long enough in the past to have a replication fork pass over position $x$.
		In the limit $\Delta x\rightarrow0$ and $\Delta t\rightarrow0$, Eq.~\ref{eq:RepFromI} becomes
		\begin{equation} \label{eq:RepFromIexp}
			f\left( x,t\right) = 1 - \exp\left[-\iint\limits_\Delta dx^\prime dt^\prime I\left( x^\prime,t^\prime\right)\right] \text{ .}
		\end{equation}
		
		Now, it is possible to define a new quantity, $g(\Delta x_p,t)$, that is a local measure of origin firing:
		\begin{equation} \label{eq:LocalOriginFiring}
			g\left(\Delta x_p,t\right) = \int\limits_{x_p}^{x_{p+1}} dx \delta\left( x-x_i\right) \int\limits_o^t dt^\prime I_i\left( t^\prime\right)
		\end{equation}
		over the region $[x_p, x_{p+1})$ of a genome of length, $L$, discretized into $M$ segments;
		\begin{align}
			\Delta x = \frac{L}{M} \qquad\qquad x_p = p\left(\Delta x\right) \qquad\qquad p = 0, 1, 2, \ldots , M-1 \text{ .}
		\end{align}
		$g(\Delta x_p,t)=0$ if there are no origins enclosed in $\Delta x_p$ because initiation will only occur at an origin.
		Thus, we replace the double integral in Eq.~\ref{eq:RepFromIexp} by the function $g(\Delta x_p,t)$ and arrive at
		\begin{equation} \label{eq:RepFromG}
			f\left( x,t\right) = 1 - \exp\left[ - \sum\limits_{p=0}^{M-1}g\left(\Delta x_p,t-\frac{\left| x-x_p \right|}{v}\right)\right] \text{ ,}
		\end{equation}
		where $v$ is the speed of replication forks, $\Delta x_p$ is the $p^\text{th}$ interval, $x_p$ is the $p^\text{th}$ position, and $\left| x-x_p \right|/v$ is the time at the edge of the past triangle in Fig.~\ref{fig:KJMA}.
		
		Recognizing that $g(\Delta x_p,t)$ represents the initiation rate of budding yeast, we can constrain it to better describe the biological system:
		First, we constrain $g$ such that replication cannot happen before the start of S phase; $g(\Delta x_p,t<0)=0$.
		Second, we constrain the initiation rate to be non-negative. Because of the definition in Eq.~\ref{eq:LocalOriginFiring}, this constrains $g$ as well: $\frac{d}{dt}g(\Delta x_p,t)\geq 0$.
		Thus, as a consequence of the first two constraints, $g(\Delta x_p,t)\geq 0$.
		
		Finally, we derive the cumulative initiation probability, $\Phi(x_p,t)$, from $g(\Delta x_p,t)$ using a calculation similar to that used for a Poisson-process~\cite{Spikes}:
		\begin{equation} \label{eq:PhiFromG}
			\Phi\left( x_p,t\right) = 1 - e^{-g\left(\Delta x_p,t\right)} \text{ .}
		\end{equation}
		The cumulative initiation distribution is an important quantity that will be revisited below, in {Sec.~\ref{sec:MIM}.
		Note that $\Phi(x_p,t)$ is a general function that can be defined throughout the genome, but in the case of budding yeast is nonzero only for values of $x_p$ that coincide with origins.


	\section{The Sigmoidal Model}
	\label{sec:SigmoidalModel}
	
	The sigmoidal model is a phenomenological approach to characterizing each origin.
	Developed by S. Yang as part of his PhD thesis, this model assumes that the functional form of $I_i(t)$ is a sigmoidal function that has a range from zero to one and that is defined by three parameters for each origin, $i$, on the genome~\cite{ScottsPaper,ScottsThesis}.
	
	Figure~\ref{fig:SigmoidalModel} shows the preliminary observations that motivated the sigmoidal model.
	First, the replicated fractin at an origin, $f(x=x_i,t)$, was extracted from experimental data of the entire genome, $f(x,t)$ (Fig.~\ref{fig:SigmoidalModel}A).
	The figure shows the analysis of microarray data~\cite{McCuneMicroArray}.
	Second, a sigmoidal curve was fit to $f(x=x_i,t)$ (Fig.~\ref{fig:SigmoidalModel}B).
	This sigmoidal curve is parameterized by the median replication time, $t_{\text{rep}}$, and by the spread of replication times, $t_{\text{width}}$:
	\begin{equation} \label{eq:SigmoidalModel}
		f(t) = {\frac{1}{1+\left({\frac{t_{\text{rep}}}{t}}\right)^r}}\text{ ,}
	\end{equation}
	where $t_{\text{width}}$ is defined by
	\begin{equation} \label{eq:SigmoidalModel2}
		t_{\text{width}} = \left(3^{1/r}-3^{-1/r}\right)t_{\text{rep}}\text{ .}
	\end{equation}
	
	\begin{figure}[tbh]
		\begin{center}
			\includegraphics[width=.82\textwidth]{Images/ScottFig3-1.pdf}
		\end{center}
			\caption[Early analysis of budding yeast]{\label{fig:SigmoidalModel} Schematic of the initial analysis of budding yeast data that led to the creation of the Sigmoidal Model.
				\textbf{A} Sample replicated fraction: smoothed data from microarray measurements of Chromosome I (solid lines)~\cite{McCuneMicroArray}.
				The black triangles indicate the locations of previously identified origins~\cite{OriginLocations}.
				Data from the replicated fraction at a single origin (grey region) at a time were analyzed.
				\textbf{B} Equation~\ref{eq:SigmoidalModel} fitted to the extracted replicated fraction at an origin.
				The fit function has parameters, $t_{\text{rep}}$ and $t_{\text{width}}$, which are shown.
				\textbf{C} Scatter plot of origin parameters from fitting the replicated fraction at every origin reveals a correlation.
				The dashed line shows $t_{\text{width}}=t_{\text{rep}}$.
				Figure reproduced with permission from S.~Yang~\cite{ScottsThesis}}
	\end{figure}
	
	One can see in Fig.~\ref{fig:SigmoidalModel}C the correlation between $t_\text{rep}$ and $t_\text{width}$.
	However, this approach ignores interactions between neighbouring origins and their effects on $f(x=x_i,t)$; these parameters may not describe intrinsic properties of the origins.
	Thus, the correlation may not be a property of the origins themselves but an accident of their geometry.
	
	To better analyze the data, Yang developed the Sigmoidal Model: an analytical method for quickly calculating the replicated fraction over the whole genome, $f(x,t)$.
	The model calculates $f(x,t)$ from a set of origins defined by three parameters ($x_i$, $t_i^{(1/2)}$, and $t^{(\text{w})}_i$), where $t_{1/2}$ and $t_\text{w}$ are defined by Eqs.~\ref{eq:SigmoidalModel} and~\ref{eq:SigmoidalModel2}~\cite{ScottsThesis}.
	With this, the entire set of experimental data was used to characterize every origin simultaneously (not illustrated).
	
	\begin{SCfigure}[1][tbh]
		\includegraphics[width=.47\textwidth]{Images/ScottTFig3-4.pdf}
		\caption[Sigmoidal model]{\label{fig:SigmoidalModel2} Scatter plot of origin parameters from fitting the replicated fraction of the entire genome with the Sigmoidal Model reveals a correlation.
			Solid points are specific origins identified for discussion in Yang's thesis.
			Figure reproduced with permission from S.~Yang~\cite{ScottsThesis}}
	\end{SCfigure}
	
	Figure~\ref{fig:SigmoidalModel2} graphs the intrinsic $t_{1/2}$ vs $t_{\text{w}}$ calculated from fitting the Sigmoidal Model to microarray data.
	Notice the strong correlation between timing width and median observed in the crude analysis of Fig.~\ref{fig:SigmoidalModel}C is present in the intrinsic parameters as well.
	This correlation means that early origins have narrowly defined firing times, while late origins have loosely defined firing times.
	An implication is that there is a mechanism that controls origin firing time that is strong at the start of S phase but weakens as S phase progresses~\cite{ScottsThesis}.
	This observation suggested the Multiple Initiator Model (MIM).
	
	
	
	\section{The Multiple Initiator Model}
	\label{sec:MIM}
	
	The MIM, in its simplest form, assumes that each origin has a given number of potential initiators that may be initiated during S phase~\cite{ScottsThesis}.
	If each of these potential initiators have equally opportunity to fire, then origins with large numbers of initiators should tend to fire earlier than origins with few initiators.
	Effectively, origins with more initiators loaded will tend to fire earlier in S phase than origins with fewer pairs.
	However, it is important to note that other factors, such as chromatin structure (the three-dimensional organization of the genome), can affect the relative firing times of origins~\cite{Chromatin}.
	
		\subsection{MIM Basics}
		\label{subsec:MIMBasics}
		
		One hypothesis for the biological mechanism that makes up a potential initiator is the MCM2-7 hexamer pairs loaded at each origin.
		During licensing, the ORC can load MCM2-7 hexamers in excess~\cite{MultiMCM}.
		A simple hypothesis (that will be discussed in Sec.~\ref{subsec:PrepModule}) is that initiators are loaded as a Poisson process:
		MCM2-7 hexamers are loaded at an origin individually with some probability determined by the affinity of that origin.
		We assume that the average number of initiators loaded at the $i^{\text{th}}$ origin, $n_i$, is a fixed quantity over different cell cycles.
		However, the actual number of initiators, $N_i$, can be different each cell cycle and is here assumed to be Poisson distributed.
		
		During S phase, the initiators are activated by the addition of Cdc45 and the GINS complex.
		The MIM assumes that each initiator has the same cumulative probability of firing as time progresses through S phase, given by
		\begin{equation}\label{eq:CPDInitiator}
			\Phi_0(t) = \frac{1}{1+\left(\frac{t^*_{1/2}}{t}\right)^{r^*}}\text{ ,}
		\end{equation}
		where $t^*_{1/2}$ is the median firing-time for a single initiator and where $r^*$ sets the width of the distribution.
		These variables are global, defining the behaviour of every initiator on the genome.
		From this assumption, the cumulative probability that an origin with $N$ loaded initiators has fired is
		\begin{equation} \label{eq:CPDEffectiveN}
			\Phi_{\text{eff}}(t,N) = 1 - \left[1 - \Phi_0(t)\right]^N\text{ .}
		\end{equation}
		
		From Eq.~\ref{eq:CPDEffectiveN}, the replicated fraction can be inferred from a set of global parameters (fork velocity, time, $t^*_{1/2}$, $r^*$, and two parameters defining the noise in the experimental data) and two parameters per origin (its position on the genome, and the number of initiators it loads).
		We start by calculating the effective cumulative firing time distribution, $\Phi_{i}^{(\text{eff})}(x,t,N_i)$ for each origin, $i$.
		Next, we invert Eq.~\ref{eq:PhiFromG},
		\begin{equation}
			\ln \left( 1- \Phi_{i}^{(\text{eff})}(x,t,N_i)\right) = - g(\Delta x_p, t) \text{ ,}
		\end{equation}
		and sum over every origin in the genome,
		\begin{equation} \label{eq:fitting}
			\sum\limits_{\text{all origins }i}\Phi_{i}^{(\text{eff})}(x,t,N_i) = - \sum\limits_{p=0}^{M-1} g(\Delta x_p,t) \text{ ,}
		\end{equation}
		to calculate the initiation rate for the entire genome.
		Finally, we can replace the exponent in Eq.~\ref{eq:RepFromG} with the left-hand side of Eq.~\ref{eq:fitting}.
		Using this process, Yang fit the parameters listed above to microarray data as part of his PhD research~\cite{ScottsThesis}.
	
		Equation~\ref{eq:CPDEffectiveN} calculates the effective cumulative probability distribution of an origin with $N$ loaded initiators.
		This is a property of a single cell, with a single value for $N$.
		In Sec.~\ref{sec:ReplicatedFraction} we mentioned that there is a subtle difference between the single-cell interpretation and the population interpretation of the replicated fraction.
		If the number of loaded initiators at an origin does not change between cell cycles (i.e. $N=n$), then the two interpretations are equivalent and Eq.~\ref{eq:CPDEffectiveN} applies to large cell populations as effectively as a single cell.
		However, if $N$ varies among cell cycles, the two interpretations diverge.
		Equation~\ref{eq:CPDEffectiveN} then becomes:
		\begin{equation} \label{eq:CPDEffectiven}
			\Phi_{\text{eff}}(t,n) = 1 - \langle\left[1 - \Phi_0(t)\right]^n\rangle\text{ ,}
		\end{equation}
		where $\langle \cdots \rangle$ denotes the ensemble average over $P_N(n)$.
		
		
		\subsection{Accounting for variability in $N$}
		\label{subsec:VariableN}
		
		Given the many factors that affect the ability of the ORC to load MCM2-7 initiators onto DNA~\cite{MultiMCM}, it is reasonable to assume that the number of initiators will vary over cell cycles.
		Thus, we need a way to evaluate Eq.~\ref{eq:CPDEffectiven} to calculate $\Phi_\text{eff}(t,n)$.
		
		We assume that initiators are loaded as a Poisson process.
		This means in a large population of cells, if a particular origin has a mean number of initiators, $n$, the standard deviation of $N$ will be $\sqrt{n}$~\cite{cowan}.
		Therefore, as $n$ grows, the relative fluctuations within the population shrinks as $n^{-1/2}$.
		Thus, for large-enough $n$, we can neglect fluctuations in $n$ and Eq.~\ref{eq:CPDEffectiveN} becomes accurate.
		However, recent experimental evidence suggests that typically, $n$ ranges between one and five, which is not large.
		
	\section{Experimental Measurement of the Number of Initiators}
	\label{sec:ExperimentsMIM}
	
	The MIM makes a strong hypothesis about the physical mechanism controlling origin firing times during S phase.
	In particular, its predictions about the relative number of MCM pairs at a given origin can be checked experimentally.
	Here, we describe recent experiments performed by Das~\emph{et~al.}, that constitute a first attempt to estimate relative and absolute numbers of loaded MCM pairs in budding yeast~\cite{Rhind}.
	
	Das~\emph{et~al.} made several measurements to test the strength of the MIM.
	First, they measured the relative number of initiators loaded at each origin; according to the MIM, origins that fire earlier should have more initiators than those that fire later.
	The relative number of initiators indicates which origins have more initiators, but cannot measure exactly how many initiators there are (i.e. it can say that origin $a$ has twice as many initiators as origin $b$ but cannot differentiate between $n_a=2$, $n_b=1$ and $n_a=200$, $n_b=100$).
	Second, Das~\emph{et~al.} measured the effect of reducing the number of initiators at an origin; according to the MIM, reducing the number of loaded initiators should delay the mean firing time of that origin.
	Third, they measured the average absolute number of initiators loaded on a particular early origin; according to the MIM, an early-firing origin should have many initiators loaded, on average.
		
		\begin{figure}[tbh!]
			\begin{center}
				\includegraphics[width=.8\textwidth]{Images/DasFig1.png}
			\end{center}
				\caption[Relative amounts of loaded MCM]{\label{fig:Das1} Results from ChIP-seq experiments measuring the relative number of loaded MCMs throughout the budding yeast genome.
					\textbf{A} ChIP-seq measurements for Chromosome X from G1 arrested wild-type cells.
					Red histogram shows uniquely located reads, grey is multiply-located reads.
					Circles along x-axis show the locations of identified origins.
					\textbf{B} Scatter plot of ChIP-seq data at origins vs their $n$ values calculated from the MIM.
					Blue origins are believed to be affected by chromatin structure~\cite{Chromatin}; green are not.
					Some origins are labeled with double circles.
					These labels refer to other parts of the data presented by Das~\emph{et al.}, but not presented here.
					ARS1 origin is labeled with text.
					Line represents the best linear fit to green dots ($r=0.56$).
					Figure reproduced with permission from N.~Rhind~\cite{Rhind}}
		\end{figure}
	
		\subsection{Relative Number of Initiators}
		\label{subsec:RelativeNo}
		
		To measure the relative number of initiators, Das~\emph{et~al.} used ChIP-seq in a population of G1-arrested cells.
		ChIP-seq (Chromatin immunoprecipitation followed by sequencing) is a technique that profiles genome-wide DNA-binding proteins, histone modifications or nucleosomes~\cite{ChIP-seq}.
		Das~\emph{et al.} used ChIP-seq to measure the number of MCMs bound to budding yeast DNA.
		In this case, Das~\emph{et~al.} prepared the experiment such that the output provided a measure of the relative number of MCM proteins loaded throughout the genome.
		Figure~\ref{fig:Das1}A shows the relative number of MCM proteins in Chromosome X of budding yeast.
		The peaks align with origins identified in OriDB~\cite{OriDB}.
		Figure~\ref{fig:Das1}B shows the relative number of MCM2-7 hexamers loaded at an origin vs.\ the $n$ value predicted from the MIM in 2010~\cite{ScottsPaper}.
		After ignoring origins that are believed to fire late due to their location in chromatin structure (blue origins in Fig.~\ref{fig:Das1}B)~\cite{Chromatin}, Das~\emph{et~al.} observed a correlation between the number of initiators and the theoretical parameter $n$ calculated by the MIM.
		This measurement confirms the first prediction made by the MIM, that the number of initiators loaded correlates with origin firing times.
		
		\begin{figure}[tbh]
			\begin{center}
				\includegraphics[width=.8\textwidth]{Images/DasFig2.png}
			\end{center}
				\caption[Replication profile of various origins]{\label{fig:Das2} Replication profiles of various origins.
					Many of the origins are outside the scope of this discussion.
					ARS1 and ARS1$\Delta$B2 are shown as the thick black and thick grey curves respectively.
					The removal of the B2 element of ARS1 causes that origin to fire, on average, 13 minutes later in S phase.
					Note that, on the y-axis, $f = [\text{Relative copy number}] - 1$.
					Figure reproduced with permission from N.~Rhind~\cite{Rhind}}
		\end{figure}
		
		
		\subsection{Suppressing the Loading of Initiators}
		\label{subsec:SuppressingInitiators}
		
		To evaluate the effect of suppressing the loading of initiators on the replication program, Das~\emph{et~al.} measured the replication profile of ARS1 and the ARS1-$\Delta$B2 mutant.
		The ARS1 origin is known to be early firing~\cite{OriDB} and should therefore have a relatively high number of initiators.
		Because the B2 element of ARS1 takes part in the recruitment of Mcm2p, the ARS1-$\Delta$B2 mutant (which has the B2 element removed) reduces MCM2-7 loading~\cite{ARS1Mutant}.
		The expectation, based on the MIM, is that the mutant ARS1-$\Delta$B2 will have a later mean firing time because of the reduced number of initiators loaded.
		By measuring the replicated fraction of cells with both wild-type and mutant ARS1 origins independently, Das~\emph{et~al.} saw a marked (13 minute) delay in the average replication timing of ARS1 caused by the $\Delta$B2 mutation (Fig.~\ref{fig:Das2}).
		
		\begin{SCfigure}[1][tbh]
			\includegraphics[width=.47\textwidth]{Images/DasFig3.png}
			\caption[Absolute number of loaded MCMs at ARS1]{\label{fig:Das3} Quantization of data from a western blot experiment that measured the amounts of MCM, ORC and Zif268 present in populations of G1 arrested plasmids and G2 arrested plasmids.
				The left-most column shows that, on average, there are about 3 initiators loaded at ARS1 during the G1 phase.
				Figure reproduced with permission from N.~Rhind~\cite{Rhind}}
		\end{SCfigure}
		
		
		\subsection{Number of Initiators Loaded}
		\label{subsec:NoInitiatorsLoaded}
		
		Das~\emph{et~al.} engineered several special plasmids and used them to measure the average number of initiators loaded at an early firing origin.
		A plasmid is a small loop of dsDNA that is separate from the genome and that is replicated independently~[\textbf{Find Source}].
		On each plasmid was engineered one of a selection of origins (plasmids were separated into populations such that each plasmid in a given population contained the same origin).
		One of the six proteins in the ORC and one of the six proteins in the MCM2-7 hexamer were tagged, so that their relative average numbers could be measured with western blotting.
		In order to calculate the absolute average numbers of MCM2-7 hexamers and ORCs, Das~\emph{et~al.} normalized the measurements using a zinc-finger.
		The normalization was possible because Zif268, the so-called ``zinc-finger,'' binds to a specific 10-bp sequence of DNA with sub-nanomolar affinity~\cite{ZincFingers}.
		By including a single instance of the Zif268 binding sequence in the plasmid, Das~\emph{et~al.} concluded that each plasmid had exactly one Zif268 protein bound to it.
		From a population of G1 phase arrested cells, the engineered plasmids were extracted and the relative number of ORCs, MCM2-7s and zinc-fingers were measured and normalized such that the average number of zinc-fingers was one.
		
		The results of Das~\emph{et~al.} for the ARS1 origin are particularly instructive.
		Figure~\ref{fig:Das3} shows the average number of MCM2-7 hexamers (one initiator is a pair of these) loaded on ARS1, and the average number of loaded ORCs during G1 arrest and G2 arrest.
		The conclusion is that there are $n\approx 3$ MCM pairs on the ARS1 origin during G1 arrest.
		It is important to restate that the MIM does not report absolute $n$ but rather gives relative $n$.
		In other words, the value for $n$ reported by the MIM  is proportional to the number of unique chances an origin has to fire.
		Thus, the fact that the number measured by Das~\emph{et~al.} does not match that predicted by the MIM is not immediately troublesome.
		However, the small number of initiators, means that cell-to-cell variability can be important.
		The question we set out to answer with this research is, thus, \emph{how does cell-to-cell variability in the initiation factor affect the replication program?}










































