\chapter{Results}
\label{ch:Results}

In this chapter, we outline our investigations into the effect of small $n$ on the Multiple Initiator Model.
Using the MIM simulator described in Ch.~\ref{ch:Methods}, we performed four major investigations.

Our preliminary investigation was a single-origin comparison between the analytical MIM and the MIM Simulator.
We defined a parameter that measures the difference between the replicated fraction from simulation and from the MIM.
In Sec.~\ref{subsec:earlywork}, our so-called ``difference parameter'' shows that small $n$ does create a disagreement between the analytical MIM and the MIM simulator proportional to $n^{-1}$.
The presence of this error and its large tail motivated further study into how small $n$ affects the MIM.

In our second investigation, our goal was to develop a new metric for measuring the error in the MIM at low $n$.
The difference parameter does not scale to more than one origin.
Section~\ref{subsec:BiasedFits} outlines the new method which consists of simulating the replicated fraction for a single origin of fixed $n$, followed by using the MIM to find the value of $n$ that best fits the simulated data.
The results of this investigation show that our first approach, while qualitatively in agreement, overestimates the difference between the MIM and the simulation.

Third, we progressed to simulating and fitting the more complex Chromosome I.
For this investigation, we started by fitting parameters with the MIM to data from DNA sequencing~\cite{StochasticTermination}.
We fit two sets of parameters by fixing $t_{1/2}$ as high and low.
In this way, we forced the MIM to produce small and high $n$ respectively.
Using the parameters from the fits, we then simulated the replication of Chromosome I.
By calculating the root-mean-squared difference between simulated data and experimental data we showed that the two simulations are effectively indistinguishable from each other.
Additionally, analysis of the fit parameters shows that the small-$n$ values are proportional to the large-$n$ values.

The surprising results from our third investigation motivated additional work that we use to explain why the MIM is inaccurate for a single origin but produces good chromosome-wide results.
We expanded our single-origin analysis to a genome with two origins and more and show that multiple origins interact to reduce the inaccuracy in inferences made by the MIM.

	\section{Single-Origin Investigations}
	\label{sec:SingleOrigin}
	
	Our early work consisted of single-origin simulations intended to motivate further study.
	In 2014, we received the results discussed in Sec.~\ref{sec:ExperimentsMIM} from N.~Rhind's lab~\cite{Rhind}.
	As mentioned, these results called into question the assumption that $n$ is large enough to ignore variations in $N$.
	Therefore, our first goal was to discover whether or not simulations of small $n$ agreed with the MIM predictions for small $n$.
	In this naive investigation, discussed further below, we developed the ``difference parameter,'' a metric to measure the difference between the simulated and predicted $f(x,t,n)$ of a single origin.
	The investigation shows that the difference parameter decreases as $n^{-1}$ and motivated the deeper research presented in this thesis.
	
	The single-origin investigations that followed our preliminary work consisted of simulating $f(n)$ and fitting the MIM parameters to the result.
	Thus, our metric changed from the difference parameter to a comparison between the simulated $n_\text{sim}$ and the fitted $n_\text{fit}$.
	With these investigations, we refined the simulation program to run more efficiently and to create data similar to that measured in sequencing experiments.
	We show that our new metric reveals a qualitatively similar behaviour for the MIM; the difference between $n_\text{sim}$ and  $n_\text{fit}$ goes approximately as $n^{-1/2}$.
	We attribute the change in the rate to the change in how the metric is defined.
		
	\begin{figure}[tbh!]
		\begin{center}
			\includegraphics[width=\textwidth]{Images/DifferenceParameterGraph.pdf}
		\end{center}
			\caption[Schematic of the Difference Parameter Calculations]{\label{fig:DifferenceParameter} Schematic of the difference parameter calculations.
				Coloured lines represent the difference parameter for different values of $n$.
				Triangles show the parameter values in the corresponding inset graphs.
				\textbf{A.} Replication fraction simulation and theory curves for $n=10$; $DP=0.037$.
				\textbf{B.} Replication fraction simulation and theory curves for $n=5$; $DP = 0.062$.
				\textbf{C.} Replication fraction simulation and theory curves for $n=1$; $DP = 0.264$.
				Also illustrated in \textbf{C} is the value $\Delta P(x)$ at the peak.
				Note that $\Delta P(x)$ is defined over the entire domain.
				}
	\end{figure}
	
	
		\subsection{The Difference Parameter}
		\label{subsec:earlywork}
		
		When starting this project, we performed a quick investigation into the difference between $f_\text{sim}(n)$ and $f_\text{MIM}(n)$ for a single origin.
		To generate $f_\text{sim}(n)$, the MIM simulator calculated the average replicated fraction for about $10^6$ sequences, producing data with very little statistical error (in contrast to the noisy data shown in Sec~\ref{sec:Noise}).
		The global parameters used for the simulation were set equal to those measured previously by fitting the MIM to microarray measurements of budding yeast~\cite{ScottsPaper}.
		The difference parameter was then defined as
		\begin{equation} \label{DifferenceParameter}
			DP = \max_{x} {\frac {\Delta P(x)} {P}} \text{ ,}
		\end{equation}
		where $\Delta P(x)$ is the difference between $f_\text{sim}(n)$ and $f_\text{MIM}(n)$ (illustrated in Fig.~\ref{fig:DifferenceParameter}), and $P$ is the peak value of $f_\text{MIM}(n)$.
		
		Figure~\ref{fig:DifferenceParameter} shows the analysis process of the difference parameter for a single origin.
		First, $f_\text{sim}(n)$ and $f_\text{MIM}(n)$ were calculated over several time steps and $n$ ranging from 1 to 128.
		Example replicated fractions are shown in the insets of Fig.~\ref{fig:DifferenceParameter}.
		From these data, we calculated $DP(n,t)$, shown in the main graph of Fig.~\ref{fig:DifferenceParameter}.
		Note the value for $DP$ ``saturates'' as $t$ increases, we call this value the ``saturated difference parameter.''
		Figure~\ref{fig:SaturatedDifferenceParameter} shows\footnote{
		The reader may notice a change in the $n$ values displayed in the graphics. Figure~\ref{fig:DifferenceParameter} the saturated difference parameter as a function of $n$ is illustrative and contains old data; accurate, but not useful.
		After producing that graph we used a slightly different set of parameters in the simulation and changed the range of $n$ simulated when producing Fig.~\ref{fig:SaturatedDifferenceParameter}}
		our measurements of the saturated difference parameter as a function of $n$.
		This initial investigation showed the saturated difference parameter decreases as $n^{-1}$.
		
		\begin{figure}[tbh]
			\begin{center}
				\includegraphics[width=0.8\textwidth]{Images/SaturatedDifferenceParameterGraph.pdf}
			\end{center}
				\caption[Saturated Difference Parameter vs. $n$]{\label{fig:SaturatedDifferenceParameter} Saturated difference parameter vs. $n$.
				The large uncertainty in low $n$ arises because the simulation time is too short to accurately measure the saturated difference parameter.
				The line is proportional to $n^{-1}$ ($\text{[Saturated Difference Parameter]} \approx \frac{0.3}{n}$).
				}
		\end{figure}
		
		From our initial investigation, we concluded that the difference parameter grows quickly with decreasing $n$.
		Therefore, we suspected that the MIM will not produce accurate results in the case that $n$ is small.
		These results motivated in-depth research into the effect of small $n$ on the MIM.
		
		
		\subsection{Biased Fits}
		\label{subsec:BiasedFits}
		
		The definition of the difference parameter does not scale to more than one origin.
		The saturated difference parameter is only measurable when the theory curve has a peak value near one.
		Additionally, the difference parameter is defined as a single value for the whole simulated genome; therefore, we cannot infer anything about more than one origin.
		Thus, we developed a second investigation that measures the bias in parameters inferred with the MIM from a simulation of small $n$.
		
		\begin{figure}[tbh]
			\begin{center}
				\includegraphics[width=0.47\textwidth]{Images/LargePopBias.pdf}
			\end{center}
			\caption[Bias in MIM fit on Large-Population Simulations]{\label{fig:LargePopulation} Scatter plot of $n_\text{sim}$ vs. $n_\text{fit}$ for simulations of a large population.
			Red circles show the data. Dashed line shows unity.
			}
		\end{figure}
		
		The process of this investigation was to calculate $f_\text{sim}(n)$ followed by fitting the parameters of the MIM to the result.
		Thus, we have two parameters: $n_\text{sim}$, the value for $n$ input to the simulation, and $n_\text{fit}$, the value for $n$ that results from the MIM fit.
		Initially, we performed these measurements using simulations of large populations of sequences (about $10^6$ and more 100 kb sequences).
		Figure~\ref{fig:LargePopulation} shows the preliminary results from our biased fit investigation on large-population simulations.
		The graph is a scatter plot of $n_\text{sim}$ vs. $n_\text{fit}$, the dashed line shows unity.
		As we expected, the bias is relatively large for low $n_\text{sim}$, but decreases as $n_\text{sim}$ grows.
		However, even using a C++ module to increase performance, these simulations were slow and were far more accurate than the current experimental standard (Sec.~\ref{sec:Noise}).
		Therefore, we limited our simulations as described in the previous chapter; in this way, we simulated data comparable to those generated experimentally.
		
		\begin{figure}[tbh]
			\begin{center}
				\includegraphics[width=0.8\textwidth]{Images/NoisyBias.pdf}
			\end{center}
				\caption[Bias in MIM Fit to Noisy Data]{\label{fig:NoisyBias} Scatter plot of $n_\text{sim}$ vs. $n_\text{fit}$ for noisy simulations of a single origin.
				Red dots show data from 50 simulations at each value $n_\text{sim}$.
				Blue circles with error bars show the mean and standard deviation of the mean.
				Dashed line shows unity.
				\textbf{Inset.} Scatter plot of the percent difference ($[n_\text{sim} - n_\text{fit}]/n_\text{sim}$) vs. $n_\text{sim}$.\\
				$\text{[Dashed line]} = 0.315/\sqrt{n_\text{sim}}$.
				}
		\end{figure}
		
		In Sec.~\ref{sec:Noise} we outlined the process used to generate noisy data.
		We used the MIM Simulator to generate $f_\text{sim}(n_\text{sim})$ with noise and used the MIM to fit the parameters to that data.
		Figure~\ref{fig:NoisyBias} is a scatter plot of $n_{sim}$ vs. the resulting $n_\text{fit}$.
		In this case, because of the increased noise in the simulated data, we performed this procedure fifty times per $n_\text{sim}$ value (red dots).
		The blue circles show the mean for each value of $n_\text{sim}$.
		Here, we see that the bias is largest for small $n_\text{sim}$, and decreases as $n_\text{sim}$ grows, in agreement with our earlier investigation.
		In the inset to Fig.~\ref{fig:NoisyBias} we show the percent difference given by $[n_\text{sim} - n_\text{fit}]/n_\text{sim}$.
		The dashed line shows $0.315/\sqrt{n_\text{sim}}$, but that trend is presented only for comparison with the difference parameter; there is either no simple fit or we are underestimating the error.
		
		The data shown in Fig.~\ref{fig:NoisyBias}, which comes from fitting the MIM to artificially noisy simulated data, shows the behaviour we expect:
		The MIM works poorly when $n$ is small, and better with increasing $n$.
		This indicates that the MIM simulator is producing good data, that is comparable to both experimental data and the analytical MIM.
		With these results assuring us the program is sound, we continued our research exploring the same process on genomes with multiple origins.
		
		
	\section{Simulations of Chromosome I}
	\label{sec:ChromosomeI}
	
	Our results from single-origin simulations indicate that the MIM does not perform well in the small-$n$ regime for a single origin.
	However, in eukaryotes, origins are not alone: as we discussed in Ch.~\ref{ch:Introduction}, replication in eukaryotes starts at many origins.
	In this section, we investigate the replicated fraction of Chromosome I and do not observe the same reduction in accuracy.
	Following our single-origin investigation, we simulated the replicated fraction of Chromosome I of budding yeast.
	For this investigation, we used the same simulation method as described above for the noisy single-origin analysis, except that the genome size and origin parameters were chosen to represent Chromosome I.
	
	\begin{table}[tbh]
		\begin{center}
			\begin{tabular}{| r | c | c | c | c | c | c |}	
				\hline
				Position (kb)	&	38.3	&	72.6	&	124.2	&	155.6	&	174	&	216	\\	\hline
				Small $n$	&	1.65	&	1.93	&	2.3	&	2.5	&	5.7	&	1.5	\\
				Large $n$	&	10.2	&	12	&	14	&	16	&	36	&	9	\\	\hline
			\end{tabular}
		\end{center}
		
		\caption[High and low $n$ fit values for Chromosome I]{\label{tab:LargeAndSmallN}
			High and low fitted values for $n$ on Chromosome I.
			Top row shows the fixed positions of the six fitted origins.
			Center row shows the values of $n$ when $t_{1/2}=40$.
			Bottom row shows $n$ from a fit with $t_{1/2}=90$.
			Small $n$ vs Large $n$ is plotted in Fig.~\ref{fig:ChrISims}
		}
	\end{table}
		
	\begin{figure}[tbh]
		\begin{center}
			\includegraphics[width=\textwidth]{Images/ChrIExpAndSim.pdf}
		\end{center}
			\caption[Experimental and Simulated Replicated Fraction of Chromosome I]{\label{fig:ThisFigure} 
				The replicated fraction of Chromosome I from experimental data~(\cite{StochasticTermination} supplementary data), and simulations with high $n$ and low $n$.
			}
	\end{figure} 

	The origin parameters were set by fitting the MIM to the replicated fraction for wild-type budding yeast reported by Hawkins~\emph{et al.}~\cite{StochasticTermination}.
	To test the effect of small $n$, the fitted parameter $t_{1/2}$ was fixed at a high value (90 minutes) to produce high values for $n$ and at a low value (40 minutes) to produce low values for $n$.
	To be sure that any effects we observed were due only to the fitted values of $n$, the origin positions were fitted once then held constant for the second fit.
	The resulting values for $n$ at each origin can be seen in Tab.~\ref{tab:LargeAndSmallN}.
		
	\begin{figure}[tbh]
		\begin{center}
			\includegraphics[width=\textwidth]{Images/RMSDiff.pdf}
		\end{center}
			\caption[Root Mean Square Difference Between Simulations and Experimental Data]{\label{fig:RMSDiff} 
				Analysis of $\Delta f_\text{rms}$.
				\textbf{A.} Plot of $\Delta f_\text{rms}(x)$ for six time-steps between high $n$ simulations and experimental data.
				The gradient goes from light grey (15 min) to black (40 min).
				\textbf{B.} Same as \textbf{A} between low $n$ simulations and experimental data.
				\textbf{C.} $\Delta f_\text{rms}$ averages over the genome vs time since the start of S phase.
			}
	\end{figure} 
		
	\begin{figure}[tbh!]
		\begin{center}
			\includegraphics[width=\textwidth]{Images/LowVsHighAndPercDiffVsX.pdf}
		\end{center}
			\caption[Low-$n$ Fit Values vs. High-$n$ Fit Values and Percentage Difference Over the Genome]{\label{fig:ChrISims}
				\textbf{A.} Scatter plot of Low-$n$ fit values vs. high-$n$ fit values.
				The line shows the best linear fit, $n_\text{low} \approx 0.16\times n_\text{high}$.
				\textbf{B.} Scatter plot of the percentage difference shown in Figs.~\ref{fig:ChrIFitVsSim}B and D vs the positions of the origins.
				Dashed line is zero.
				Labels show $n_\text{sim}$.
			}
	\end{figure} 
	
	We simulated the two sets of parameters that resulted from the high-$n$ and low-$n$ fits.
	The resulting replicated fractions are shown in Fig.~\ref{fig:ThisFigure}.
	To quantify the quality of the two fits, we calculated the root mean squared difference $\Delta f_\text{rms}$ between each replicated fraction and the experimental data.
	We simulated each set of parameters fifty times and averaged $\Delta f_\text{rms}$ at each time step over the fifty simulations.
	Figures~\ref{fig:RMSDiff}A and B show $\Delta f_\text{rms}(x,t)$ for high and low $n$ respectively and Fig.~\ref{fig:RMSDiff}C shows the average value for each time step, $\Delta f_\text{rms}(t)$.
	Surprisingly, and in contrast to the results presented so far, these data imply that the quality of the MIM fits is nearly identical\footnote{
	We observe a trend for high-$n$ simulation to be slightly more accurate than low-$n$ simulations; however, the error bars show the standard deviation of the mean of 50 simulations. Thus, the trend is well within the noise of a single measurement.}
	for large and small $n$.
		
	Further, we were interested in the relationship between the large-$n$ values from the fit and the small-$n$ values from the fit.
	The MIM does not claim that $n$ is the absolute number of initiators on an origin; rather, the fitted parameter should be proportional to the absolute number of initiators.
	If this is true, the values for low-$n$ and high-$n$ should be linearly related.
	Our suspicion, based on the results of our single-origin investigation, was that the relationship would not be linear but go approximately as $n^{-1/2}$.
	However, our results from fitting to Chromosome I seem to contradict this suspicion.
	Indeed, Fig.~\ref{fig:ChrISims}A shows that the relationship is linear: $n_\text{small} \propto n_\text{large}$.
		
	\begin{figure}[tbh!]
		\begin{center}
			\includegraphics[width=\textwidth]{Images/ChrIFitVsSim.pdf}
		\end{center}
			\caption[Chromosome I $n_\text{fit}$ vs. $n_\text{sim}$ and Percentage Difference]{\label{fig:ChrIFitVsSim}
				$n_\text{fit}$ vs. $n_\text{sim}$ and percentage difference for high $n$ and low $n$ simulations of Chromosome I.
				\textbf{A.}~Scatter plot of $n_\text{fit}$ vs. $n_\text{sim}$ for high $n$.
				Dashed line is unity.
				\textbf{B.}  Scatter plot of the percentage difference between $n_\text{fit}$ and $n_\text{sim}$ vs. $n_\text{sim}$ for high $n$.
				Dashed line is zero.
				\textbf{C - D.} Same as \textbf{A} and \textbf{B} respectively for low $n$.
			}
	\end{figure} 
	
	Using the same technique as outlined in Sec.\ref{subsec:BiasedFits}, we used the MIM to calculate $n_\text{fit}$ values for each origin that we simulated on Chromosome I.
	For the six origins we simulated, we calculated the percent difference between $n_\text{fit}$ and $n_\text{sim}$ (shown in Figs.~\ref{fig:ChrIFitVsSim} B and D).
	The trend we observe in the single origin case of decreasing percentage difference with increasing $n$ is no longer present.
	Our measurements of Chromosome I indicate that the MIM is approximately equally effective for both high-$n$ and low-$n$.
	
	We suspected that the spatial organization of the origins in Chromosome I played a role in the observed equality in the fit.
	Therefore, Fig.~\ref{fig:ChrISims}B shows the percentage difference as a function of location within the genome.
	We do not observe any meaningful pattern in this plot.
	
	
	\section{Neighbouring Origins Reduce the effect of Small $n$}
	\label{sec: NearOrigins}
	
	The results shown in Sec.~\ref{sec:ChromosomeI} appear to contradict the results from the previous, single-origin investigations.
	Therefore, we ask the question: \emph{What is different between the single-origin simulations and the simulations of Chromosome I?}
	The immediate answer is that the number of origins has changed from one to six.
	Perhaps then, multiple origins cooperate somehow to reduce the effect of small $n$ on the predictive power of the MIM.
	Here, we show our investigations of genomes with more than one origin.
	
		\subsection{Two-Origin Investigation}
		\label{subsec:TwoOrigins}
		
		To test our hypothesis that multiple origins cooperate to increase the efficacy of the MIM in the small-$n$ regime, we expanded our single-origin investigation to a genome with two origins.
		With the addition of a second origin, there is a new consideration: By what distance should the two origins be separated?
		The obvious maximum distance the origins should be separated is $2vt_\text{sim}^\text{(max)}$, because if they are any further apart, their replicated regions will never overlap.
		For our simulations that is 180 kb.
		However, it is very rare for an origin to fire at the start of S phase, so placing the two origins 180 kb apart is not wise.
		We looked to the fitted locations of origins on Chromosome I shown in Tab.~\ref{tab:LargeAndSmallN} as a guide.
		Here, we see the two closest origins are 18.4 kb separated, and the two furthest origins are 54.2 kb separated.
		Therefore, we simulated two origins with equal $n_\text{sim}$ spaced both 18.4 kb apart and 52.4 kb apart.
		
	\begin{figure}[tbh]
		\begin{center}
			\includegraphics[width=\textwidth]{Images/TwoOriginPercDiff.pdf}
		\end{center}
			\caption[Scatter Plots of Single-and Two-Origin Percent Difference]{\label{fig:TwoOrigins} 
				Scatter plots of single-and two-origin percent difference between $n_\text{fit}$ and $n_\text{sim}$.
				\textbf{A.} Percent difference vs. $n_\text{sim}$ for a single origin.
				\textbf{B.} Same as \textbf{A} for two near origins (18.4 kb separation)
				\textbf{C.} Same as \textbf{A} for two distant origins (54.2 kb separation)
			}
	\end{figure} 
		
		Figure~\ref{fig:TwoOrigins} shows the percent difference between $n_\text{sim}$ and $n_\text{fit}$ resulting from these simulations for $n$ ranging from 2 to 64 compared to single-origins simulations.
		Contrary to our hypothesis, these results show that two origins are fitted less accurately than single origins.
		
		
		\subsection{Multiple-Origin Investigation}
		
		The results so far have been surprisingly contradictory:
		In the single-origin case, the MIM fits get progressively worse as $n$ decreases.
		In the many-origin case of Chromosome I, the MIM fits are equally accurate for both high $n$ and low $n$.
		In the two-origin case, the MIM fits are less accurate than the single-origin case for low $n$.
		
	\begin{figure}[tbh]
		\begin{center}
			\includegraphics[width=0.8\textwidth]{Images/ChromosomeI.png}
		\end{center}
			\caption[Sketch of Sub-sequences of Chromosome I]{\label{fig:ChrISubSeq} 
				Sketch of sub-sequences of Chromosome I used in simulations.
				Vertical lines denote chosen edge points for sub-sequences.
				Circles represent origins.
				Horizontal lines represent sub-sequences (number of origins labeled and the full chromosome (labeled).
				Star indicates the sub-sequence highlighted in Fig.~\ref{fig:IncreasingOrigins}.
			}
	\end{figure} 

		We would like to see a transition from the inaccurate 2-origin system to the accurate 6-origin case.
		To accomplish this, we chose to simulate sub-sequences of Chromosome I containing 2 origins, 3 origins, and 4 origins.
		Figure~\ref{fig:ChrISubSeq} illustrates how the sub-sequences were selected with divisions occurring directly in the middle\footnote{
		Due to a calculation error, the second division was 4 kb off the midpoint. This should not have a strong impact on the results.}
		of neighbouring origins.
		There may be better selection criteria for where the endpoints of sub-sequences should fall;
		for example, the fifth origin has higher $n$ than both of its neighbours; therefore, it will have a larger region of influence than they.
		However, as the results will show, in addition to its simplicity, this method is effective.
		
	\begin{figure}[tbh]
		\begin{center}
			\includegraphics[width=0.8\textwidth]{Images/MultipleOrigins.pdf}
		\end{center}
			\caption[Scatter Plots of $n_\text{fit}$ vs. $n_\text{sim}$ for Two, Three and Four Origins]{\label{fig:IncreasingOrigins} 
				Scatter Plots of $n_\text{fit}$ vs. $n_\text{sim}$ for two, three and four origins for high $n$ and low $n$.
				Open circles show low-$n$ data.
				Black circles show high-$n$ data.
				Dotted lines in right-most graphs show a single sub-sequence of four origins.
				Dashed lines show unity.
			}
	\end{figure} 
		
		We simulated the sub-sequences of Chromosome I using the $n$ values shown in Tab.~\ref{tab:LargeAndSmallN}.
		Figure~\ref{fig:IncreasingOrigins} shows a scatter plot of $n_\text{fit}$ vs. $n_\text{sim}$ for two-origin (left), three-origin (middle), and four-origin (right) sub-sequences (labeled with a star in Fig.~\ref{fig:ChrISubSeq}) of Chromosome I in the high-$n$ (black circles) and low-$n$ (open circles) regimes.
		The two dotted lines show the four origins on a single sub-sequence, in order.
		In this data we observe two trends:
		First, as the number of origins in the sequence increases the MIM fits grow in accuracy.
		While between high $n$ and low $n$, the accuracy is still better in the high-$n$ regime, we suspect that as $n$ grows the difference will become negligible.
		Second, as illustrated by the dotted lines in Fig.~\ref{fig:IncreasingOrigins}, we observe that origins in the center of the sequence (surrounded by origins) are fitted more accurately than origins on the edges that have a single neighbour only.
		These two trends appear to work together to make the MIM accurate for all values of $n$ on sequences with many origins.
		





































